bitnami/shared/kyverno/kafka-client-policies.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-kafka-certificate
+  annotations:
+    policies.kyverno.io/title: Create certificate for Kafka client
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/subject: Deployment, Statefulset
+    policies.kyverno.io/description: >-
+      This policy creates the kafka client certificate for the annotated targets.
+spec:
+  # Generate for existing targets
+  generateExisting: true
+  background: true
+  rules:
+    - name: generate-msk-certificate
+      match:
+        any:
+          - resources:
+              annotations:
+                uw.systems/msk-kafka-client: "*"
+              kinds:
+                - Deployment
+                - StatefulSet
```

Comment:
Can we expect kafka client CronJobs?

Reply:
I specifically left that out, as I don't see a use case for it. If it is one out there, they can do it the old way.